packages/server-side-graphql-client/tests/index.test.js

```diff
@@ -0,0 +1,224 @@
+const { Text, Integer } = require('@keystonejs/fields');
+const { multiAdapterRunners, setupServer } = require('@keystonejs/test-utils');
+
+const {
+  createItems,
+  createItem,
+  deleteItem,
+  deleteItems,
+  getItem,
+  getItems,
+  updateItem,
+  updateItems,
+} = require('../lib/server-side-graphql-client');
+
+const testData = [
+  {
+    data: {
+      name: 'test',
+      age: 30,
+    },
+  },
+  { data: { name: 'test2', age: 40 } },
```

Comment:
Looks weird having one of these as a one-liner and the other spread over 4 lines.

Reply:
I'm open to condensing it but don't want to do it :)